universal-application-tool-0.0.1/conf/evolutions/default/21.sql

```diff
@@ -0,0 +1,11 @@
+# --- Create a table that links programs to the program admin accounts that administer them.
+
+# --- !Ups
+create table program_accounts (
+  program_id bigint not null,
+  account_id bigint not null,
+  primary key (program_id, account_id)
```

Comment:
You probably want a foreign key constraint here.

Reply:
(plus ebean wants these to be plural)

Reply:
Thanks! Added the constraint